src/components/common/RTF.tsx

```diff
@@ -0,0 +1,33 @@
+import ReactMarkdown from "react-markdown";
+import remarkYext, {
+  yextRemarkRehypeHandlers,
+} from "src/components/common/rtf-library/index";
+// import { Link } from "@yext/sites-components";
+
+/**
+ * This component is intended for rendering RTF v1 ("Legacy") fields.
+ *
+ * If you are trying to render a "Markdown" field, you can use react-markdown
+ * directly without needing the additional plugins provided here.
+ *
+ * If you are trying to render a "v2" rich text field, import the LexicalRichText
+ * component from \@yext/react-components.
```

Comment:
Do you think we could detect the v2 format and call that component fromn within this function so that developers don't have to worry about the field type?